test/wpt/runner/runner.mjs

```diff
@@ -224,14 +226,27 @@ export class WPTRunner extends EventEmitter {
             signal: AbortSignal.timeout(timeout)
           })
 
-          console.log(colors(`[${finishedFiles}/${total}] PASSED - ${test}`, 'green'))
+          if (result?.subtests.every((subtest) => subtest.status === 'PASS')) {
+            this.#stats.testsPassed += 1
+            console.log(colors(`[${finishedFiles}/${total}] PASSED - ${test}`, 'green'))
+          } else {
+            this.#stats.testsFailed += 1
+            console.log(colors(`[${finishedFiles}/${total}] FAILED - ${test}`, 'red'))
```

Comment:
Tests can have expected failures, marking the entire file as failing isn't the correct behavior here.

Reply:
Hi @KhafraDev and thank you for your review! Sorry, I hadn't considered this scenario. I'll try and fix that 😁